modules/admin_manual/pages/appliance/configuration/office.adoc

```diff
@@ -168,6 +168,30 @@ Here are the required steps:
 * Select on the App name
 * Upgrade the App
 
+== Registering your Enterprise Edition
+
+If you bought the Enterprise Edition and received the **license.lic** file, you need to import it to unlock the enterprise features.
+
+When you are a Debian based Linux distributive user, the license.lic file must be placed in the following folder:
+
+----
+/var/www/onlyoffice/Data/license.lic
+----
+
+Now you will have access to the enterprise features of onlyoffice.
+
+Additional information can be found https://helpcenter.onlyoffice.com/installation/docs-enterprise-install-ubuntu.aspx[in the onlyoffice documentation]
```

Comment:
```suggestion
== Registering your ONLYOFFICE Enterprise Edition

If you purchased the ONLYOFFICE Enterprise Edition and received the **license.lic** file, you need to import it to unlock the enterprise features.

When you are a Debian based Linux distribution user, the license.lic file must be placed into the following folder:

----
/var/www/onlyoffice/Data/license.lic
----

Doing so, you will have access to the enterprise features of ONLYOFFICE.

Additional information can be found in the https://helpcenter.onlyoffice.com/installation/docs-enterprise-install-ubuntu.aspx[ONLYOFFICE Documentation]
```

Reply:
ONLYOFFICE is a proper name and needs to be written in capital letters as it is done on their homepage

Reply:
agreed